custom/icds_reports/utils/aggregation_helpers/distributed/awc_infrastructure.py

```diff
@@ -23,6 +23,10 @@ class AwcInfrastructureAggregationHelper(StateBasedAggregationPartitionedHelper)
         'preschool_kit_available', 'preschool_kit_usable'
     )
 
+    @property
+    def temp_ucr_tablename(self):
```

Comment:
And either way is there a name that might be more descriptive that could be used? e.g. `get_table_name_for_previous_months_agg` (not 100% sure that's right)

Reply:
maybe `get_ucr_tablename_from_previous_months_agg` but yeah. i will think on some of this naming stuff overnight and try to improve these